pw-3/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pw-3",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
type: module

Reply:
Changed type of project to "module" and "start" command had been rewritten